### DISCUSSION TITLE
SEB through Moodle won't start

### DISCUSSION THREAD

**Discussion by NikolaasD** [SafeExamBrowser/seb-win-refactoring#291] [General]
Hello everyone

**Problem**
For some students, SEB will not start when they click 'start safe exam browser' in a moodle quiz.
We are using the manual config in Moodle, where Moodle generates the SEB file and starts the SEB browser with this file.

This is what I see in the Moodle quiz logs:
_The user with id '2082' has been prevented from accessing quiz with id '6183' by the Safe Exam Browser access plugin. The reason was 'Ongeldige SEB configuratie sleutel.'. Expected config key: '85ce3a1f6e0804c4e2ecda3d2156725dcba2d8d72baa43982e777d9d69bc6717'. Received config key: ''. Received browser exam key: ''._

Ongeldige SEB configuratie sleutel = invalid SEB configuration key
![image](https://user-images.githubusercontent.com/74493214/146240377-1f427f4d-c081-4399-a475-73f19b7c058b.png)

**Situation**
I've got this problem with, on average, 20 out of 550 students. The problem wasn't visible during the regular lesson period, because quizes were organized per class. Right now we're doing exams with up to 200 students at the same time.

**Background information**
- SEB version 2.4
- 600 identical laptops, same batch from factory. So, hardware is as identical as it can be.
- Image provided by external company. SEB is included in image.
- Windows Defender as AV on laptop
- Fortigate as network firewall

**I've tried**

- I contacted our Moodle host. They say that this is a problem with the configuration of the laptops. 
- I contacted our external laptop partner, since they provide us with the image. They say VC++ sometimes doesn't get installed the right way when deploying the image. I've been able to test manually installing this on one computer. No succes.
- I've tried removing and installing the latest version of the SEB. No succes.
- I've reimaged the computer: about 70% succes rate.

I would be very grateful if someone could explain what this config key is about. Why would this continue to be a problem after reinstalling the SEB?
Any other advice on what to look at, do or test is also greatly apreciated.

Many thanks for developing and maintaining the SEB.

**Comment by NikolaasD** [comment]
Hello

I think I've found my own answer.

Removing 2.4 en installing 3.3.1.388 seems to work. Me previous tests were older than this release.

**Reply by dbuechel** [reply]
Yes, we highly recommend updating from 2.x to the latest official release version of 3.x. The configuration key enables an exam system or LMS to verify that a user is actually using the intended configuration, see https://safeexambrowser.org/windows/win_usermanual_en.html#ExamPane or https://safeexambrowser.org/developer/seb-config-key.html for more information.
